payments/forms.py

```diff
@@ -26,69 +26,8 @@ class Meta:
 
 
 class PaymentMethodForm(forms.Form):
-    payment_type = forms.ChoiceField(choices=PaymentTypeField.CHOICES)
-    source_id = forms.CharField(widget=forms.HiddenInput(), required=False)
-    card_token = forms.CharField(widget=forms.HiddenInput(), required=False)
-
-    def clean(self) -> Any:
-        cleaned_data = self.cleaned_data
-
-        # extract source id
-        if "source_id" in cleaned_data:
-            source_id = cleaned_data["source_id"]
-        else:
-            source_id = None
-        cleaned_data["source_id"] = source_id
-        source_is_blank = source_id == "" or source_id is None
-
-        # extract card id
-        if "card_token" in cleaned_data:
-            card_token = cleaned_data["card_token"]
-        else:
-            card_token = None
-        cleaned_data["card_token"] = card_token
-        card_is_blank = card_token == "" or card_token is None
-
-        if source_is_blank and card_is_blank:
-            raise forms.ValidationError(
-                "Either a Source ID or a Card Token must be given"
-            )
-
-        if (not source_is_blank) and (not card_is_blank):
-            raise forms.ValidationError(
-                "Exactly one of Source ID and Card Token must be given"
-            )
-
-        return cleaned_data
-
-    def attach_to_customer(self, customer: str) -> [Optional[bool], Optional[str]]:
-        source_id = self.cleaned_data["source_id"]
-        token = self.cleaned_data["card_token"]
-        return stripewrapper.update_payment_method(customer, source_id, token)
+    pass
 
 
 class CancellablePaymentMethodForm(PaymentMethodForm):
```

Comment:
Is there a reason for this class? Both it and the parent class are empty. Wouldn't one be enough?